packages/build/src/plugins_core/functions/zisi.js

```diff
@@ -14,8 +14,7 @@ export const getZisiParameters = ({
   repositoryRoot,
 }) => {
   const nodeVersion = childEnv.AWS_LAMBDA_JS_RUNTIME
-  const isManifestEnabled = isRunningLocally || featureFlags.buildbot_create_functions_manifest === true
```

Comment:
Just to make sure, do we intend to run this on local builds?
https://github.com/netlify/build/pull/3378 originally excluded those with `isRunningLocally`.

Reply:
I don't think we would be needing this anymore.

Reply:
> #3378 originally excluded those with `isRunningLocally`.

@ehmicky Are you sure? I think we were always running this on local builds (https://github.com/netlify/build/pull/3378/files#diff-61d09d0c00742d9d5c5ae1614b6b7ed37b6f28b32f7d72670fb33026c36c6237R47).

Reply:
Oh, right!
I was raising just to make sure we double check removing that condition, but if it looks good to both of you, this looks good to me too. :+1: